packages/apolloschurchapp/src/tabs/discover/DiscoverFeed/getContentChannels.js

```diff
@@ -4,7 +4,7 @@ import { CONTENT_ITEM_FRAGMENT } from 'apolloschurchapp/src/content-single/getCo
 import { LARGE_CARD_FRAGMENT } from 'apolloschurchapp/src/ui/ContentCardConnected';
 
 export default gql`
-  query getContentChannels {
+  query getContentChannel {
```

Comment:
Not sure why we need this change.

```suggestion
  query getContentChannels {
```

Reply:
Yeah, me neither. I didn't mean to change that!